cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -654,10 +698,12 @@ def _add_fsx_storage(self, id: str, shared_fsx: BaseSharedFsx):
                 file_system_type=LUSTRE,
                 storage_type=shared_fsx.fsx_storage_type,
                 subnet_ids=self.config.compute_subnet_ids,
-                security_group_ids=self._get_compute_security_groups(),
+                security_group_ids=[sg.ref for sg in file_system_security_groups],
```

Comment:
Why this change was made? After the change, if a user would avoid ParallelCluster creating any security groups, besides setting `SecurityGroups` in `Queues`, they have to avoid using any managed EFS or FSx.